addons/l10n_ar_withholding/models/l10n_ar_partner_tax.py

```diff
@@ -0,0 +1,39 @@
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+import logging
+# from dateutil.relativedelta import relativedelta
+_logger = logging.getLogger(__name__)
+
+
+class L10nArPartnerTax(models.Model):
+    _name = "l10n_ar.partner.tax"
+    _description = "Argentinean Partner Taxes"
+    _order = "to_date desc, from_date desc, tax_id"
+
+    partner_id = fields.Many2one(
+        'res.partner',
+        required=True,
+        ondelete='cascade',
+    )
+    tax_id = fields.Many2one(
+        'account.tax',
+        # domain=[('applicability', '=', 'taxes')],
```

Comment:
```suggestion
        # domain=[('applicability', '=', 'taxes')],
```

Reply:
the suggested change is to delete the comment?

Reply:
To clean out...